Hue.xcodeproj/project.pbxproj

```diff
@@ -611,6 +618,8 @@
 				PRODUCT_NAME = Hue;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```

Comment:
Is there a reason behind setting the `SWIFT_OPTIMIZATION_LEVEL` to `-Owholemodule` ? :)